load/duplex_channel.go

```diff
@@ -9,11 +9,11 @@ type duplexChannel struct {
 	toScanner chan bool
 }
 
-// newDuplexChannel returns a duplexChannel whose buffer sizes are equal to queue
-func newDuplexChannel(queue int) *duplexChannel {
+// createDuplexChannel returns a duplexChannel with specified buffer sizes
```

Comment:
Name change seems unnecessary to me; I prefer `new` to `create` here (`NewFoo`/`newFoo` seems pretty standard golang convention)

Reply:
Ok, agreed. Code-digging artifact and indeed, `newXXX()` is used all over the project, so there is no need to introduce any `createXXX()`
Summary: updated